src/shared/bg_pmove.cpp

```diff
@@ -4516,6 +4516,21 @@ PmoveSingle
 
 ================
 */
+// set the firing flag for continuous beam weapons
+static void setFireBeam( buttonNumber_t btn )
+{
+	if ( !( pm->ps->pm_flags & PMF_RESPAWNED ) && pm->ps->pm_type != PM_INTERMISSION &&
+			usercmdButtonPressed( pm->cmd.buttons, BTN_ATTACK ) &&
```

Comment:
This function doesn't use its argument
```suggestion
			usercmdButtonPressed( pm->cmd.buttons, btn ) &&
```

Reply:
good catch!